src/ert/config/queue_config.py

```diff
@@ -111,6 +115,28 @@ def create_local_copy(self) -> QueueConfig:
         )
 
 
+def _validate_lsf_options(lsf_options: List[Tuple[str, str]]) -> None:
+    # Checks currently only for overwriting of 'QUEUE_OPTIONS LSF LSF_SERVER'
+    keywords_to_check = ["LSF_SERVER"]
+    for keyword in keywords_to_check:
```

Comment:
There are very many different `QUEUE_OPTION` for the different `QUEUE_SYSTEM` available.
Would it be an idea to test if there are _ANY_ duplicates here? The end result would be the same regardless.
The function could also cover other systems, i.e. TORQUE, LOCAL, SLURM etc.

I guess we can omit `MAX_RUNNING` as we already know this might show up multiple times.